LEGO1/legoinputmanager.h

```diff
@@ -27,9 +29,9 @@ class LegoInputManager
 
   virtual long Tickle() override; // vtable+0x8
 
-  char m_pad00[0x19C];
+  undefined m_pad00[0x15c];
```

Comment:
```suggestion
  undefined m_pad40[0x15c];
```

Reply:
m_20 and m_40 are MxCriticalSection

Reply:
m_20 is part of `MxPresenter`. I didn't verify m_40.